packages/tradershub/src/components/CurrencySwitcher/CurrencySwitcherAccountInfo.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { Fragment } from 'react';
+import { IconToCurrencyMapperType } from 'src/constants/constants';
+import { THooks } from 'src/types';
+import { Text } from '@deriv/quill-design';
```

Comment:
@aizad-deriv just one question, we still use `Text` from `quill-design` or need to use it from `@deriv-com/ui`?

Reply:
Yeah I think it's better to use it from `@deriv-com/ui` rather than `quill-design`